cmds/modules/provisiond/api.go

```diff
@@ -71,6 +73,9 @@ func setupGPURmb(router rmb.Router, store provision.Storage) {
 		return gpus, nil
 	}
 	gpus.WithHandler("list", func(ctx context.Context, payload []byte) (interface{}, error) {
+		if kernel.GetParams().IsGPUDisabled() {
+			return nil, fmt.Errorf("gpu is disabled on this node")
```

Comment:
Why return an error? this can just return an empty list. (no gpus)

Reply:
The client doesn't need to know if the farmer disable the GPU feature, all what he needs to know is that the node has no GPUs for renting